daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -103,6 +103,9 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
         choice Archive (self) (arg: Unit): Unit,
           controllers Cons @Party [M:T {signatory} this] (Nil @Party)
           to upure @Unit (TRACE @Unit "archive" ());
+        choice @nonConsuming Divulge (self) (divulgee: Party): Unit,
+          controllers Cons @Party [divulgee] (Nil @Party)
+          to upure @Unit ();
```

Comment:
review this file is probably simpler if you look  at  [cdd8d21f](https://github.com/digital-asset/daml/pull/14206/commits/cdd8d21fe886a6d94a82b6bb4d87f7975f9428ee#diff-440a32292a8dee9e4f1170183801055319a8e027dcd378995d532736d6ee012b)